### PR TITLE
Add language support.

### PR DESCRIPTION
**Pull request by ahmadazimi**
Now users can pass desired language to detect font loaded.

```javascript
FontDetect.onFontLoaded(fontname, callback [, onFail: xxx, {msInterval: yyy, msTimeout: zzz, lang:'fa'}]);
```

### CODE REVIEW

**Comment by JenniferSimonds** [comment]
This is an excellent idea! See the comments for some small changes (naming of variables, mostly, and a data check). Then we can merge this.

**Comment by ahmadazimi** [comment]
Hi Jennifer,

First of all, Happy New Year to you.

I made the fixes you mentioned in above comments. If everything is okay, please merge them.

File: lib/fontdetect.js [hunk 1]
```diff
@@ -101,6 +108,7 @@ FontDetect = function()
             // Our hashtable of optional params.
             var msInterval = (p_options && p_options.msInterval) ? p_options.msInterval : 100;
             var msTimeout  = (p_options && p_options.msTimeout) ? p_options.msTimeout : 2000;
+			eLang = (p_options && p_options.lang && p_options.lang in aChars) ? p_options.lang : 'en';
```
Comment (JenniferSimonds) [inline lib/fontdetect.js]: p_options.lang needs to be changed as well.

eLang = (p_options && p_options.eLang && p_options.eLang in aChars) ? p_options.eLang : 'en';

**Comment by JenniferSimonds** [comment]
Thanks, Ahmad; it looks like one line still needs to be changed.

**Comment by ahmadazimi** [comment]
I've change that line and also I've added eLang section in README,md

**Pull request merged**